### PR TITLE
Restore banz scraper and lawgit functionality

### PR DESCRIPTION
**Pull request by JBBgameich**
GitHub is not updating the checks status of the old pull request, so maybe this one works better.

### CODE REVIEW

**Comment by JBBgameich** [comment]
Is it okay if I merge this myself to be able to work on stuff building on top of this? This code has been around for a few days, but initially in a different pull request.

**Comment by ulfgebhardt** [comment]
Since @darkdragon-001 does not do the review, I will approve and rely on the tests.
Would you like me to disable the required review to make life easier for you? This was a messure to make merging easier when we had no tests since any person could approve your PR and continue

**Review by ulfgebhardt — APPROVED** [review]

**Comment by JBBgameich** [comment]
If there is nobody who likes to do the reviews right now, I think disabling the requirement would be good.

The tests are currently only testing the syntax and not the functionality, so reviews would generally be helpful.

**Comment by ulfgebhardt** [comment]
I think my capabilities in reviewing python are less then the automatic syntax check ;)

I disabled the review requirement, so you just need to pass the tests. It seems like you are the only one willing & able to contribute at the moment :(

File: lawgit.py [hunk 1]
```diff
@@ -21,9 +23,15 @@
...
 from collections import defaultdict
 
-from git import Repo
+from git import Repo, Commit, DiffIndex, Diff
 from git.exc import GitCommandError
 
+from typing import List, Dict, Tuple
+
+
+def log(*message: str):
```
Comment (darkdragon-001) [inline lawgit.py:32]: Should we use [`logging`](https://docs.python.org/3/library/logging.html)?

Comment (ulfgebhardt) [reply lawgit.py:32]: your choice

**Pull request merged**